client/src/templates/Introduction/intro.css

```
@@ -8,11 +8,40 @@
   overflow-wrap: break-word;
 }
 
+.block-title-wrapper {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+}
+
 .big-block-title {
   font-size: 1.5rem;
   overflow-wrap: break-word;
 }
 
+.block-title-translation-cta {
+  white-space: nowrap;
+  padding: 0.2em 0.5em;
+  color: var(--highlight-color);
+  background-color: var(--highlight-background);
+}
+
+.block-title-translation-cta:hover,
+.block-title-translation-cta:focus {
+  text-decoration: none;
+  padding: 0.2em 0.5em;
+  color: var(--highlight-background);
+  background-color: var(--highlight-color);
+}
+
+.block-cta-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  padding-right: 15px;
+  padding-top: 25px;
```

```suggestion
  justify-content: center;
  padding-right: 15px;
```
@ahmadabdolsaheb couple of things: I got a warning about `start` from Gatsby and don't we just want it to be centered? It looks better to me centered.

---

Sure, we could either place it in accordance with card or the title. Both are alright with me.

---

I will give our Accent Colors a round of audit. That could come in another pr.